conda/recipes/cuspatial/meta.yaml

```
@@ -77,7 +77,7 @@ requirements:
     - cudf ={{ minor_version }}
     - pylibcudf ={{ minor_version }}
     - geopandas >=0.11.0
-    - numpy >=1.23,<2.0a0
+    - numpy >=2.1.0,<3.0a0
```

```suggestion
    - numpy >=2.0.0,<3.0a0
```
Well, that was silly of me. We can't use 2.1 yet (due to numba), so this needed adjustment.  The wheel tests seem to be fine, let's hope that this fixes the conda ones.
(After which, need to undo any other shenanigans.)